src/reducers/images.js

```diff
@@ -64,6 +66,14 @@ const images = (state = initialState, { type, payload }) => {
         }
       };
 
+    case UPDATE_GIF_FILENAME:
+      return {
+        ...state,
+        ...{
```

Comment:
Question for @ctlusto - is there a reason why you prefer spreading out a new object here, rather than just adding keys directly to the return object? e.g. a reason for returning

```js
{
  ...state,
  ...{
    gifFileName: payload.gifFileName
  }
}
```

instead of returning

```js
{
  ...state,
  gifFileName: payload.gifFileName
}
```

?

Reply:
Uh...no. That's probably leftover from something I was doing earlier.